docs/Tutorial_MERLIN.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>Here are a couple of suggestions:</p><p> </p><ol><li>Use <code>T</code> instead of <code>T_A</code></li><li>The z-norm for <code>T_i</code> can be moved outside of the <code>for j</code> loop</li><li>Instead of returning <code>failure</code> , I think it is fine to simply <code>return C</code>  with length <code>0</code>  <span class="ql-cursor">﻿</span></li></ol><p> </p><p>It also looks like we may be able to parallelize the inner <code>for j</code> loop</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/418/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
> It also looks like we may be able to parallelize the inner `for j` loop

I need to read a few articles to see how this can be done.

Reply:
Don't worry about it for now. Let's focus on getting it to work first before making it fast

Reply:
<p>I have experience with this and can help assist. We'll be using the Numba package for parallelization</p>